ui/app/mainui/AppMain.qml

```diff
@@ -29,15 +29,15 @@ import shared.popups.send 1.0
 import shared.popups.send.views 1.0
 import shared.stores.send 1.0
 
+import StatusQ 0.1
+import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
```

Comment:
Import this as `SQUtils` instead, as done elsewhere

Reply:
To be done